metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -244,26 +259,18 @@ def process_schemas(self, catalog: proxy.Catalog) -> Iterable[MetadataWorkUnit]:
 
     def process_tables(self, schema: proxy.Schema) -> Iterable[MetadataWorkUnit]:
         for table in self.unity_catalog_api_proxy.tables(schema=schema):
-            filter_table_name = (
-                f"{table.schema.catalog.name}.{table.schema.name}.{table.name}"
-            )
-
-            if not self.config.table_pattern.allowed(filter_table_name):
+            if not self.config.table_pattern.allowed(table.ref.qualified_table_name):
```

Comment:
the filter_table_name isn't quite the fully qualified table name, since it doesn't include the metastore name

Reply:
Yeah, so I made `TableReference.qualified_table_name` not include metastore name, while `TableReference.__str__` includes metastore name. I'm realizing that naming can be confusing though. Any ideas on a better one?